Docs/Book/Cookbook.rst

```diff
@@ -1011,10 +1405,53 @@ Organometallics with Dative Bonds
    CN(C)(C)->[Pt]
 
 
+Enumerate SMILES
```

Comment:
This demonstrates an out-of-date way of doing this.
Since the 2018.09 release it's been possible to pass the `doRandom` argument to `MolToSmiles()` in order to get a more random SMILES for a molecule.
So now you can just do: `Chem.MolToSmiles(m,doRandom=True,canonical=False)`

Reply:
Here's a somewhat more up-to-date thread you could reference: https://www.mail-archive.com/rdkit-discuss@lists.sourceforge.net/msg08684.html

Reply:
Awesome, thanks for pointing that out, I missed this new feature in the mailing list.